src/components/Table/GridTable.tsx

```diff
@@ -284,9 +293,11 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
       const RowComponent = observeRows ? ObservedGridRow : MemoizedGridRow;
 
       return (
-        <GridCollapseContext.Provider value={row.kind === "header" ? collapseAllContext : collapseRowContext}>
+        <GridCollapseContext.Provider
+          key={`${row.kind}-${row.id}`}
```

Comment:
@chr1sjf0x just adding this here to resolve the React console error.

Reply:
TY 🚀